src/main/java/easy/mind/com/api/service/Impl/UserServiceImpl.java

```diff
@@ -56,4 +57,9 @@ public List<UserDTO> getAll() {
                 .map(UserDtoToUser::convert)
                 .collect(Collectors.toList());
     }
+
+    @Override
+    public boolean throwIfNotExist(long id) {
+        return readById(id) == null;
```

Comment:
readById never returns null. It returns object or throws.
Here you don't need to call readById. Repository has default method repo.exists(id). Use it here and throw exception if does not exists.